pkg/apis/networking/v1alpha1/ingress_validation.go

```diff
@@ -87,10 +87,12 @@ func (h HTTPIngressPath) Validate(ctx context.Context) *apis.FieldError {
 		return apis.ErrMissingField(apis.CurrentField)
 	}
 	var all *apis.FieldError
-	// Must provide as least one split.
-	if len(h.Splits) == 0 {
-		all = all.Also(apis.ErrMissingField("splits"))
-	} else {
+	switch {
+	case len(h.Splits) == 0 && h.RewriteHost == "":
+		all = all.Also(apis.ErrMissingOneOf("splits", "rewriteHost"))
+	case len(h.Splits) != 0 && h.RewriteHost != "":
+		all = all.Also(apis.ErrMultipleOneOf("splits", "rewriteHost"))
```

Comment:
Can you please document this requirement in the API doc?

Reply:
👍 done.